.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
 
       - name: Install system dependencies
         run: |
-          brew install gnu-time opam gtksourceview3 adwaita-icon-theme
+          brew install gnu-time opam gtksourceview3 adwaita-icon-theme flock
```

Comment:
?

Reply:
See the changes in makefile and the PR description, we lock calls to dune as it is not reentrant (yet).